pyanaconda/ui/gui/spokes/lib/refresh.py

```diff
@@ -25,8 +25,8 @@
 from pyanaconda.threading import threadMgr, AnacondaThread
 from pyanaconda.ui.gui import GUIObject
 from pyanaconda import constants
+from pyanaconda.storage import storage_initialize
 
-from blivet.osinstall import storage_initialize
 
 __all__ = ["RefreshDialog"]
 
```

Comment:
On the line 94, I guess that there should be `self.storage.protected_dev_names`, otherwise it fails.